### PR TITLE
python3-ubi - add python version env var

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
Ready

## Description
Add the `PYTHON_VERSION` env var to the python3-ubi image.

### CODE REVIEW

File: docker/python3-ubi/Dockerfile [hunk 1]
```diff
@@ -98,6 +98,8 @@ Options = UnsafeLegacyRenegotiation\n' > /tmp/ssl.cnf \
 RUN groupadd -g 4000 demisto && \
     useradd -u 4000 -g demisto demisto -s /bin/sh
 
+ENV PYTHON_VERSION=$(python3 -c 'import platform; print(platform.python_version())')
```
Comment (JudahSchwartz) [inline docker/python3-ubi/Dockerfile:101]: does this work? thought it cant be dynamic.
I think we should do ARG PYTHON_VERDION=3.10.8 that we use for this variable and also for the download and install iteslf. Or maybe use ENV if the env cant depend on the arg?

Comment (GuyAfik) [reply docker/python3-ubi/Dockerfile]: you are right it can't be dynamic

File: docker/python3-ubi/Dockerfile [hunk 2]
```diff
@@ -19,11 +19,13 @@ RUN dnf upgrade -y --nodocs && \
     dnf clean all && \
     rm -rf /var/cache/dnf
 
-# install python3
-RUN wget https://www.python.org/ftp/python/3.10.8/Python-3.10.8.tgz && \
+ARG PYTHON_VER=3.10.8
```
Comment (JudahSchwartz) [inline docker/python3-ubi/Dockerfile:22]: any way to declare as an ENV here and use it on line 27 so we dont need to declare 3.10.8 twice? 
Id also use a symlink to the dynamic_version script we have in python3 so we dont need to repeat 3.10.8 in the build conf either

**Review by JudahSchwartz — APPROVED** [review]
Approved except my last comment. See if thats doable, not sure it is

**Comment by JudahSchwartz** [comment]
also I would add to changelog that you added this env var

**Comment by GuyAfik** [comment]
> also I would add to changelog that you added this env var

the change log is only for py3-native

File: docker/python3-ubi/Dockerfile (hunk 2 above)
Comment (GuyAfik) [reply docker/python3-ubi/Dockerfile:22]: we use multistage, is it possible? regarding dynamic_version this can be done

Comment (GuyAfik) [reply docker/python3-ubi/Dockerfile]: added dynamic version symlink

**Pull request merged**